test/io/jesi/backpack/macros_test.cljc

```diff
@@ -163,6 +163,7 @@
       (is (= (set vals) -all))))
 
   (testing "allows function composition"
+    (ns-unmap 'io.jesi.backpack.macros-test '-all)
```

Comment:
remove any existing definitions for -all

Reply:
set by defconsts

Reply:
I tried to use fixtures and a fn to prevent the duplicate definition.. The warning came back...